torch/_inductor/kernel/mm.py

```diff
@@ -1135,12 +1139,14 @@ def tuned_scaled_mm(
                 )
 
         for config in scaled_mm_configs(m, n, k):
-            if k <= 16:
-                continue  # Triton crashes in this case
+            if guard_or_false(k <= 16):
+                # Triton crashes however uncommon for real workloads
+                continue
 
             # On NVIDIA B200 GPUs, K dim must be >= 32 for tcgen05.mma.kind::f8f6f4.* PTX instruction to be valid
             # source: https://docs.nvidia.com/cuda/parallel-thread-execution/#tcgen05-matrix-shape
-            if using_b200() and k < 32:
+            if using_b200() and guard_or_false(k < 32):
```

Comment:
do we need to add a torch.check here?
form the comment above seems like k must be >32?
torch.check(k>=32)?